nodetypes/radon.nodes.aws/AwsApiGateway/files/configure/configure.yml

```diff
@@ -1,9 +1,16 @@
 ---
 - hosts: all
+  vars:
+    api_gw_stage: "production"
```

Comment:
You could also expose this var as an input parameter that uses `get_property`, which reads from a property definition `stage` (having `deafult` set to `production`). WDYT?

Reply:
If that has the same outcome, I am open to add that freedom to make that property customizable.
I am not sure whether I understood the details correctly to replicate that in the code though.

Reply:
No worries, this can also be easily added later when really needed ;-)